examples/nextjs/package.json

```diff
@@ -17,7 +17,9 @@
     "@types/react-dom": "18.2.4",
     "next": "13.4.1",
     "react": "18.2.0",
-    "react-dom": "18.2.0",
-    "typescript": "^4.9.4"
+    "react-dom": "18.2.0"
+  },
+  "devDependencies": {
+    "typescript": "^5.3.0-dev.20231014"
```

Comment:
I have to wait for the merge until the vscode release bundled with typescript 5.3.